src/shogun2-webapp-archetype/src/main/resources/archetype-resources/src/main/webapp/WEB-INF/__artifactId__-servlet.xml

```diff
@@ -47,6 +47,12 @@ ${shogun2-parent-package}.*.web" )
 	</beans:bean>
 
 	<annotation-config />
+	
+	<!-- Use the full path within the current servlet context to find an appropriate handler.
+		s. http://docs.spring.io/spring/docs/4.0.7.RELEASE/spring-framework-reference/htmlsingle/#mvc-handlermapping -->
+	<beans:bean id="handlerMapping" class="org.springframework.web.servlet.mvc.method.annotation.RequestMappingHandlerMapping">
+		<beans:property name="alwaysUseFullPath" value="true"/>
```

Comment:
This could be useful for the more complex projects using multiple controllers in its context, but you're right, for the archetype that would be enough to use default config (`alwaysUseFullPath=false`).